plugins/modules/ocapi_command.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ocapi_command
+short_description: Manages Out-Of-Band controllers using Open Composable API (OCAPI).
```

Comment:
```suggestion
short_description: Manages Out-Of-Band controllers using Open Composable API (OCAPI)
```
`short_description` should never end with a period (as opposed to other descriptions).